src/cmd/create.go

```diff
@@ -502,6 +553,23 @@ func createHelp(cmd *cobra.Command, args []string) {
 	}
 }
 
+// ensureImageName translates an image URI into a "pullable" reference
+//
+// On error is returned an empty string
+func ensureImageName(image, release string) string {
```

Comment:
Umm... why not call it `ensureFullyQualifiedImage` or something like that? Otherwise, it's a bit confusing as to why we would ensure an image name when we already have `image`.

Reply:
Yes, sure. `ensureFullyQualifiedImage` sounds better.